react/features/recording/components/Recording/styles.web.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
I get it because I've worked on mobile a lot. Could you, @hristoterezov, please put an XXX comment why it's necessary?